sys/fuchsia/msi.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <zircon/syscalls.h>
+include <zircon/types.h>
+
+resource zx_msi[zx_handle]
+
+msi_allocate(handle zx_msi, count flags[msi_count], out_allocation ptr[out, zx_msi])
```

Comment:
If the `handle zx_msi` is optional here (presumably since where does one get the first handle), then it should be declared as `handle zx_msi[opt]`.

Reply:
Good point! Thanks, updated.